From 4df441f7a38bf7bdbc625cfd7b79c7b9f0780adc Mon Sep 17 00:00:00 2001
From: Amos Kong <akong@redhat.com>
Date: Thu, 21 Nov 2013 04:58:50 +0100
Subject: [PATCH 06/14] doc: fix hardcoded helper path

RH-Author: Amos Kong <akong@redhat.com>
Message-id: <1385009930-5005-1-git-send-email-akong@redhat.com>
Patchwork-id: 55827
O-Subject: [RHEL-7.0 qemu-kvm PATCH v2] doc: fix hardcoded helper path
Bugzilla: 1016952
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>
RH-Acked-by: Vlad Yasevich <vyasevic@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

Bugzilla: 1016952
Brew: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=6621357
Upstream: merged, we should parse C macros in texi mode in future

The install directory of qemu-bridge-helper is configurable,
but we use a fixed path in the documentation.

DEFAULT_BRIDGE_HELPER macro isn't available in texi mode,
we should always use "/path/to/" prefix for dynamic paths
(e.g.: /path/to/image, /path/to/linux, etc).

Signed-off-by: Amos Kong <akong@redhat.com>
Reviewed-by: Stefan Hajnoczi <stefanha@redhat.com>
Signed-off-by: Michael S. Tsirkin <mst@redhat.com>
(cherry picked from commit 420508fbba2a6e8eaff008715b5f7eff83f8e865)
---
v2: replace hardcode path prefix by '/path/to/'
---
 qemu-options.hx |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qemu-options.hx |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/qemu-options.hx b/qemu-options.hx
index 3c60140..e3c5d67 100644
--- a/qemu-options.hx
+++ b/qemu-options.hx
@@ -1593,7 +1593,7 @@ to disable script execution.
 
 If running QEMU as an unprivileged user, use the network helper
 @var{helper} to configure the TAP interface. The default network
-helper executable is @file{/usr/local/libexec/qemu-bridge-helper}.
+helper executable is @file{/path/to/qemu-bridge-helper}.
 
 @option{fd}=@var{h} can be used to specify the handle of an already
 opened host TAP interface.
@@ -1617,7 +1617,7 @@ qemu-system-i386 linux.img \
 #launch a QEMU instance with the default network helper to
 #connect a TAP device to bridge br0
 qemu-system-i386 linux.img \
-                 -net nic -net tap,"helper=/usr/local/libexec/qemu-bridge-helper"
+                 -net nic -net tap,"helper=/path/to/qemu-bridge-helper"
 @end example
 
 @item -netdev bridge,id=@var{id}[,br=@var{bridge}][,helper=@var{helper}]
@@ -1626,7 +1626,7 @@ Connect a host TAP network interface to a host bridge device.
 
 Use the network helper @var{helper} to configure the TAP interface and
 attach it to the bridge. The default network helper executable is
-@file{/usr/local/libexec/qemu-bridge-helper} and the default bridge
+@file{/path/to/qemu-bridge-helper} and the default bridge
 device is @file{br0}.
 
 Examples:
-- 
1.7.1

